packages/gcc7.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Gcc7 < Package
+  description 'The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, and Go.'
+  homepage 'https://www.gnu.org/software/gcc/'
+  version '7.3.0'
+
+  source_url 'https://github.com/gcc-mirror/gcc/archive/gcc-7_3_0-release.zip'
+
+  source_sha256 'd8cb199f0d98bd83e3dac645126722e44137c24cbe5e87e1113dd4270f7530c6'
+
+  depends_on 'buildessential'
+  depends_on 'binutils'
+  depends_on 'gmp'
+  depends_on 'mpfr'
+  depends_on 'mpc'
+  depends_on 'isl'
+  depends_on 'cloog'
+  depends_on 'glibc'
+  depends_on 'gawk'
```

Comment:
`depends_on 'unzip' => :build`?

Reply:
Ah yeah good catch.  What does the `=>` syntax mean?  This file is the extent of my Ruby knowledge.

Reply:
It just means it's a build instead of run-time dependency.  You don't need to include any dependencies that are already in Chromebrew "core".  See https://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txt.